packages/vuetify/src/components/VCombobox/VCombobox.tsx

```diff
@@ -156,12 +157,20 @@ export const VCombobox = genericComponent<new <
         }
 
         if (!val) selectionIndex.value = -1
-        if (isFocused.value) menu.value = true
 
         isPristine.value = !val
       },
     })
     watch(_search, value => {
+      if (cleared.value) {
+        // Implement open-on-clear (https://github.com/vuetifyjs/vuetify/issues/16925)
```

Comment:
Just add a single comment to better record its original purpose for future reference

Reply:
This is what git is for. We typically only put links here if it references an outside resource.